sdk/loadtestservice/Azure.Developer.LoadTesting/src/LoadTestAdministrationClient.cs

```diff
@@ -131,213 +47,41 @@ public virtual Response UploadTestFile(string testId, string fileId, string file
-        /// <example>
-        /// This sample shows how to call UploadTestFile with required parameters and request content, and how to parse the result.
-        /// <code><![CDATA[
-        /// var credential = new DefaultAzureCredential();
-        /// var client = new LoadTestAdministrationClient("<https://my-service.azure.com>", credential);
-        /// var data = File.OpenRead("<filePath>");
-        /// Response response = client.UploadTestFile("<testId>", "<fileId>", RequestContent.Create(data));
-        /// JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
-        /// Console.WriteLine(result.ToString());
-        /// ]]></code>
-        /// This sample shows how to call UploadTestFile with all parameters and request content, and how to parse the result.
-        /// <code><![CDATA[
-        /// var credential = new DefaultAzureCredential();
-        /// var client = new LoadTestAdministrationClient("<https://my-service.azure.com>", credential);
-        /// var data = File.OpenRead("<filePath>");
-        /// Response response = client.UploadTestFile("<testId>", "<fileId>", RequestContent.Create(data), 1234);
-        /// JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
-        /// Console.WriteLine(result.GetProperty("url").ToString());
-        /// Console.WriteLine(result.GetProperty("fileId").ToString());
-        /// Console.WriteLine(result.GetProperty("filename").ToString());
-        /// Console.WriteLine(result.GetProperty("fileType").ToString());
-        /// Console.WriteLine(result.GetProperty("expireTime").ToString());
-        /// Console.WriteLine(result.GetProperty("validationStatus").ToString());
-        /// ]]></code>
-        /// </example>
-        /// <remarks>
-        /// Below is the JSON schema for the response payload.
-        /// Response Body:
-        /// Schema for <c>FileUrl</c>:
-        /// <code>{
-        ///   url: string, # Optional. File URL.
-        ///   fileId: string, # Optional. File unique identifier.
-        ///   filename: string, # Optional. Name of the file.
-        ///   fileType: &quot;0&quot; | &quot;1&quot; | &quot;2&quot;, # Optional. Integer representation of the file type (0 = JMX_FILE, 1 = USER_PROPERTIES, 2 = ADDITIONAL_ARTIFACTS)
-        ///   expireTime: string (ISO 8601 Format), # Optional. Expiry time of the file
-        ///   validationStatus: string, # Optional. Validation status of the file
-        /// }
-        /// </code>
-        /// </remarks>
-        public virtual async Task<Response> UploadTestFileAsync(string testId, string fileId, string fileName, RequestContent content, int? fileType = null, RequestContext context = null)
+        public virtual async Task<FileUploadOperation> BeginUploadTestFileAsync(string testId, string fileName, RequestContent content, WaitUntil waitUntil = WaitUntil.Started, string fileType = null, RequestContext context = null)
```

Comment:
- Method must be called `UploadTestFileAsync`.
- `WaitUntil waitUntil` must be the first parameter in the method and should never has a default value